README.rst

```diff
@@ -38,41 +27,41 @@ Installation through pip is recommended::
     $ pip install cassandra-driver
 
 For more complete installation instructions, see the
-`installation guide <http://datastax.github.io/python-driver/installation.html>`_.
+`installation guide <http://scylladb.github.io/python-driver/installation.html>`_.
```

Comment:
We should be able to use relative path here and elsewhere </installation>

Reply:
No we can't, it's not the same domain